oxenstored/perms.ml

```diff
@@ -65,7 +65,7 @@ module Node = struct
 
   let perm_of_string s =
     let ty = permty_of_char s.[0]
-    and id = int_of_string (String.sub s 1 (String.length s - 1)) in
+    and id = Utils.int_of_string_exn (String.sub s 1 (String.length s - 1)) in
```

Comment:
This could probably be implemented with sscanf - parsing the char and number in one go.